### ISSUE TITLE
Move this repository to the archive

### ISSUE THREAD

**Issue by sh0rtener** [AmadeusWM/dotfiles-hyprland#25]
Your config is deprecated. I can fix some bugs if you wish

**Comment by AmadeusWM** [comment]
True, I'm not maintaining it anymore. If you are willing to fix some bugs, I'll gladly accept any PR :) but I don't use Arch anymore, so I won't be able to test the changes tho

**Comment by sh0rtener** [comment]
Well, then maybe move the repository to the archive, because my friend wrote to me in the middle of the night that he had a lot of errors on the whole screen)

**Comment by AmadeusWM** [comment]
alright, thank u for reminding me
